src/Events/FeatureValueDeleted.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Laravel\Pennant\Events;
+
+use Illuminate\Queue\SerializesModels;
+
+class FeatureValueDeleted
```

Comment:
```suggestion
class FeatureDeleted
```

Matching `FeatureUpdated`